packages/node-terra/src/indexer/fetchterra.service.test.ts

```diff
@@ -83,7 +83,7 @@ describe('FetchService', () => {
     const loopPromise = fetchService.startLoop(6684553);
     // eslint-disable-next-line @typescript-eslint/require-await
     fetchService.register(async (content) => {
-      if (Number(content.block.block.header.height) === 6684563) {
+      if (Number(content.block.block.block.header.height) === 6684563) {
```

Comment:
Is this a mistake? Same thing in `fetchterra.service.ts`, make me think it is due to a global replace.

Reply:
No, at first TerraBlockContent was something like:

```
{
   block: BlockInfo
   ...
}
```

Now It's changed to:
```
{
   block: TerraBlock
   ...
}
```

Where TerraBlock is:

```
{
   block: BlockInfo,
   txs: TxInfo[]
}
```